.travis.yml

```diff
@@ -7,13 +7,13 @@ services:
 matrix:
   fast_finish: true
   include:
-    - env: DRUPAL=8.7.x
+    - env: DRUPAL_INSTALL="drush dl drupal-8.7.x --drupal-project-rename=drupal" UPDATE_PHPUNIT="composer run-script drupal-phpunit-upgrade"
       php: 7.1
-    - env: DRUPAL=8.7.x
+    - env: DRUPAL_INSTALL="drush dl drupal-8.7.x --drupal-project-rename=drupal" UPDATE_PHPUNIT="composer run-script drupal-phpunit-upgrade"
       php: 7.2
-    - env: DRUPAL=8.8.x
+    - env: DRUPAL_INSTALL="composer create-project drupal/legacy-project:8.8.0-beta1 drupal" UPDATE_PHPUNIT=""
```

Comment:
would be great to update to rc1 https://www.drupal.org/project/drupal/releases/8.8.0-rc1

Reply:
Done.